editor-tool-highlighting/editor-tool-highlighting-ui/src/main/resources/SyntaxHighlighting/EnableSyntaxHighlighting.xml

```diff
@@ -179,6 +199,143 @@ require(['jquery', '$xwiki.getURL('SyntaxHighlighting.SyntaxHighlighting', 'jsx'
   }
 });
 
+(function() {
```

Comment:
It would have been nice to have a comment at this point explaining the code separation since otherwise it looks a bit weird (and like a mistake).